app/controllers/posts_controller.rb

```diff
@@ -1,5 +1,10 @@
 class PostsController < ApplicationController
-  def index; end
+  def index
+    @user = User.find(params[:user_id])
+    @posts = @user.posts.all
+  end
 
-  def show; end
+  def show
+    @post = Post.find(params[:id])
+  end
```

Comment:
- [ ] I am getting an error when I try to create a post please fix that or share how you are adding your data in the README incase I am missing something
![post](https://user-images.githubusercontent.com/44745855/142284494-1d005231-810c-4df2-98dd-fd7d5f16d9ca.png)